### DISCUSSION TITLE
How to get to the searched text if it is not in a headline

### DISCUSSION THREAD

**Discussion by Ypot** [alphapapa/org-ql#387] [Q&A] [answered]
When I do a search for a text that is in the body of an entry, I get in the results just the headline that contains that text. Then, I have to go to the result, and do a new "isearch" to get to the place where it is placed the text.

How could be org-ql be configured to show and to go to the exact searched text in the body of the entries?

Bests regards

**Comment by alphapapa** [comment, marked as answer]
This is an interesting idea.  Something could be implemented to parse the plain-text tokens out of a query and search for one of them after showing a result entry.  It would be somewhat of a non-trivial feature to implement.  Feel free to file a feature request.

**Reply by Ypot** [reply]
Thanks, I will. 
You already got it with your rifle!
